docs/Makefile

```diff
@@ -2,7 +2,7 @@ all: public
 
 -include resources.mk
 
-public: content/*.md static/**/* ${RESOURCES} content/license.md static/images/**/*.png
+public: content/commands content/*.md static/**/* ${RESOURCES} content/license.md static/images/**/*.png
```

Comment:
it looks like everything in `static/images/**/*.png` is also taken care of with `static/**/*`
I know this is not a change you made for this PR, but something I noticed.